docs/src/rust/getting-started/expressions.rs

```diff
@@ -42,7 +42,26 @@ fn main() -> Result<(), Box<dyn std::error::Error>>{
     // --8<-- [end:exclude]
 
     // --8<-- [start:filter]
-    // TODO
+    let start_date = NaiveDate::from_ymd_opt(2022, 12, 2)
+        .unwrap()
+        .and_hms_opt(0, 0, 0)
+        .unwrap();
+    let end_date = NaiveDate::from_ymd_opt(2022, 12, 8)
+        .unwrap()
+        .and_hms_opt(0, 0, 0)
+        .unwrap();
+    let out = df
+        .clone()
+        .lazy()
+        .filter(
+            col("c")
+                .dt()
+                .datetime()
```

Comment:
You can directly compare the column. No need to go via the `dt().datetime()` conversion.